app/src/androidTest/java/com/android/streetworkapp/ui/parkoverview/AddEventTest.kt

```diff
@@ -121,4 +121,40 @@ class AddEventTest {
     composeTestRule.onNodeWithTag("addEventScreen").assertIsDisplayed()
     composeTestRule.onNodeWithTag("addEventButton").assertIsDisplayed()
   }
+
+  @Test
+  fun addEventScreenWithUnchangedTimeDoesNotChangeScreen() {
```

Comment:
Is it easy for you to add a test that tests when the event title is empty (check the actual SonarCloud report)? With this, you could expect 100% line coverage, but if it's too complex, don't worry, the coverage is already quite high.

Reply:
I agree with you. This is the point of the second test, but SonarCloud does not agree. (On the jaccoco report in the other hand, the lines are considered covered, I am a bit lost with this). Since the coverage is high enough in the whole project and in this PR, I think it is better to stay on this coverage.

Reply:
No problem then, thank you for the clarification!